quisby.py

```diff
@@ -215,6 +222,18 @@ def data_handler():
                         ret_val = extract_coremark_data(path, system_name, OS_RELEASE)
                         if ret_val:
                             results += ret_val
+                    elif test_name == "coremark_pro":
+                        ret_val = extract_coremark_pro_data(path, system_name, OS_RELEASE)
+                        if ret_val:
+                            results += ret_val
+                    elif test_name == "passmark":
+                        ret_val = extract_passmark_data(path, system_name, OS_RELEASE)
+                        if ret_val:
+                            results += ret_val
+                    elif test_name == "pyperf":
+                        ret_val = extract_pyperf_data(path, system_name, OS_RELEASE)
+                        if ret_val:
+                            results += ret_val
                     else:
                         continue
```

Comment:
We should spit out a warning if the test is not recognized.

Reply:
Ok david.